RELEASES.md

```diff
@@ -6,11 +6,11 @@ Since Envoy AI Gateway depends on the Envoy Gateway and Envoy Proxy, we will fol
 In other words, we aim to cut the release of the Envoy AI Gateway a few days or a week after the new version of the Envoy Gateway
 is released. Therefore, the release cycle of the Envoy AI Gateway will be approximately every 2-3 months.
 
-We do not distinguish between major and patch releases. We will increment the minor version number by one for each release
-except that we will cut the v1.0.0 release when we have a first stable control plane API, i.e. the introduction of
+We do not distinguish between major and minor versions. We will increment the minor version number by one for each release
```

Comment:
I think major version can still be incremented when we have major feature or architecture change after the 1.x.x stable release ?

Reply:
good point - will make it that way

Reply:
done